operations/commit_queue.go

```diff
@@ -213,15 +213,21 @@ func listCommitQueue(ctx context.Context, client client.Communicator, ac *legacy
 	grip.Infof("Queue Length: %d\n", len(cq.Queue))
 	for i, item := range cq.Queue {
 		grip.Infof("%d:", i)
-		author, _ := client.GetCommitQueueItemAuthor(ctx, projectID, restModel.FromStringPtr(item.Issue))
```

Comment:
I'm sorry: I think I made a mistake.
What if the queue type is PR (and there's no patch yet because it's not at the head of the queue)?

Reply:
i was thinking about this. We don't really _need_ the author in the list though: people know what they're committing so the summary should be enough to see which one is there's, rather than making a whole call to get it. It's up to you though!